### DISCUSSION TITLE
Getting property in Blade component by property name

### DISCUSSION THREAD

**Discussion by mnabialek** [livewire/livewire#4900] [1. Help]
So I have simple Livewire component with property let's say foo. In the view I use Blade component using syntax like this:

```
<x-admin.form.color-input wire-model="foo" class="max-w-lg" />
```

In this Blade component I have input:

```
<input type="text" name="{{ $wireModel }}" id="{{ $wireModel }}" wire:model.debounce.500ms="{{ $wireModel }}" 
```

that's working fine.

But the problem is how to really display value of property named $wireModel (so foo in this case) inside this Blade component - I mean here just display it on page without any input?

I know I could pass value directly to component but I don't want to do this. The only way I found is:

```
{{ $this->getPropertyValue($wireModel) }}
```

but it's pretty ugly. Is there any other way to do this?

**Comment by joshhanley** [comment]
@mnabialek I use `$this->getPropertyValue()` too.

Hope this helps!
